databases/week2/school_database.sql

```diff
@@ -0,0 +1,38 @@
+CREATE database school_data;
+USE school_data;
+SET NAMES utf8mb4;
+
+CREATE TABLE `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `begins` DATETIME NULL DEFAULT NULL,
```

Comment:
a `DATE` type here would be enough since you probably don't need to know the time